src/main/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtil.java

```diff
@@ -95,18 +95,11 @@ private JavadocUtil() {
      */
     public static JavadocTags getJavadocTags(TextBlock textBlock,
             JavadocTagType tagType) {
-        final boolean getBlockTags = tagType == JavadocTagType.ALL
-                                         || tagType == JavadocTagType.BLOCK;
-        final boolean getInlineTags = tagType == JavadocTagType.ALL
-                                          || tagType == JavadocTagType.INLINE;
-
         final List<TagInfo> tags = new ArrayList<>();
-
-        if (getBlockTags) {
+        if (tagType == JavadocTagType.ALL || tagType == JavadocTagType.BLOCK) {
             tags.addAll(BlockTagUtil.extractBlockTags(textBlock.getText()));
         }
-
-        if (getInlineTags) {
+        if (tagType == JavadocTagType.ALL || tagType == JavadocTagType.INLINE) {
```

Comment:
> src/main/java/com/puppycrawl/tools/checkstyle/utils
JavadocUtil.java (1) getJavadocTags() Scope of variable getInlineTags is too broad

this fix is controversial.
if demand variable to be moved close to IF , I am agree.

Inlining expressions is not very good as it variable help us to avoid comment over IF. But I agree that name of variable should be changed to `isInlineTags`  same for other variable.

Reply:
Done.